snippets/service-account-auto-secret-removed.adoc

```diff
@@ -0,0 +1,17 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[id="auto-generated-sa-token-secrets_{context}"]
+== About automatically-generated service account token secrets
+
+In {product-version}, {product-title} is adopting an link:https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md#urgent-upgrade-notes-1[enhancement from upstream Kubernetes] where the `LegacyServiceAccountTokenNoAutoGeneration` feature is enabled by default. As a result, when creating new serivce accounts (SA), a service account token secret is no longer automatically genereated. Previously, {product-title} automatically added a service account token to a secret for each new SA. 
```

Comment:
You could remove the passive voice at the end with "where the ... feature is enabled by default" to "which enables the ... feature by default"